checkov/terraform/graph_builder/utils.py

```diff
@@ -26,9 +26,10 @@
 def is_local_path(root_dir: str, source: str) -> bool:
     # https://www.terraform.io/docs/modules/sources.html#local-paths
     return (
-        source.startswith("./")
-        or source.startswith("/./")
-        or source.startswith("../")
+        source.startswith(r"./")
+        or source.startswith(r"/./")
+        or source.startswith(r"../")
+        or source in os.listdir(root_dir)
```

Comment:
can you remove this line, you accidentally added it back again

Reply:
good catch, on it

Reply:
waiting now on your approve to merge :)